packages/design-system-react/src/components/temp-components/Maskicon/Maskicon.types.ts

```diff
@@ -3,15 +3,20 @@ import type { ComponentProps } from 'react';
 /**
  * Maskicon component props.
  */
-export type MaskiconProps = ComponentProps<'div'> & {
+export type MaskiconProps = Omit<ComponentProps<'img'>, 'width' | 'height'> & {
```

Comment:
Updating to img tag type and not allowing for overrides of width and height attributes